inc/issue.class.php

```diff
@@ -359,7 +380,96 @@ static function getReopenableStatusArray() {
    }
 
    static function getAllStatusArray($withmetaforsearch=false) {
-      return Ticket::getAllStatusArray($withmetaforsearch);
+      $ticket_status = Ticket::getAllStatusArray($withmetaforsearch);
+      $form_status = array('waiting', 'accepted', 'refused');
+      $form_status = array_combine($form_status, $form_status);
+      $all_status = array_merge($ticket_status, $form_status);
```

Comment:
There is an issue using array_merge here : numeric keys are reindexed from 0 and this triggers a PHP warning line 464. (displayed with debug mode enabled)

Using the array operator + solves the issue.